model/converter/json/from_domain.go

```diff
@@ -22,18 +22,30 @@ package json
 
 import (
 	"github.com/uber/jaeger/model"
-
 	"github.com/uber/jaeger/model/json"
 )
 
 // FromDomain converts model.Trace into json.Trace format.
 // It assumes that the domain model is valid, namely that all enums
 // have valid values, so that it does not need to check for errors.
 func FromDomain(trace *model.Trace) *json.Trace {
-	return fromDomain{}.fromDomain(trace)
+	fd := fromDomain{}
+	fd.convertKeyValuesFunc = fd.convertKeyValues
+	return fd.fromDomain(trace)
+}
+
+// FromDomainES converts model.Span into json.Span format.
+// This is separate from the FromDomain above, because we do not store
+// Traces in Elastic Search -- only Spans.
+func FromDomainES(span *model.Span) *json.Span {
```

Comment:
As I commented earlier, we're introducing a couple of options that affect how the converter is working
  * whether it generates `processID` (the default behavior) or simply embeds the `process`
  * how it deals with the `parentID` (the default behavior is to only use references)

So I would like the code and API to be expressed in the ^^^ terms, rather than referencing ES. ES is just where you use the new features, not what those features are.